test/e2e/scenario/logs.scenario.ts

```diff
@@ -78,6 +78,8 @@ describe('logs', () => {
     .withLogs({ forwardErrorsToLogs: true })
     .run(async ({ serverEvents, baseUrl, servers }) => {
       await browserExecuteAsync((done) => {
+        // This example formerly used "response.body!.getReader().cancel()" to cancel the stream midway.
+        // However, as we not longer support ".cancel()" in streams and thus why we use "response.blob()".
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏
does using blob cancel the stream? 
if not, why are we using `response.blob` and not calling `done` directly after the fetch resolve?
any reason to use `blob` and not `json` or `text` instead?

Reply:
> does using blob cancel the stream?

Good point, no. I will update the comment.

> why are we using response.blob and not calling done directly after the fetch resolve?

I initially had issues getting this condition to work because we were aborting the call too early to be able to fetch the first couple bytes.

I assumed calling `done` would also abort the stream too early for us to collect the bytes, but that does not seem to be the case. I will therefore not use any response methods (`blob`,`json` and `text`) and simply use `done`

> any reason to use blob and not json or text instead?

No, we do not care about the contents of the response (also we are not going to use any of these methods).